_episodes/048-herbert-wolverson.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Game Development with Herbert Wolverson"
+date: 2021-11-26T18:25:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e048-herbert-wolverson.mp3
+duration: "1:05:00"
+length: "63273551"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Herbert Wolverson](https://twitter.com/herberticus), the author of the book [Hands-on Rust](https://pragprog.com/titles/hwrust/hands-on-rust/).
+
+
+## Contributing to Rustacean Station
+
+Rustacean Station is a community project; get in touch with us if you'd like to suggest an idea for an episode or offer your services as a host or audio editor!
+
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps 
+- [@0:57] - About Herbert and his [book](https://pragprog.com/titles/hwrust/hands-on-rust/)
+- [@3:01] - Explaining Rust's [traits](https://doc.rust-lang.org/book/ch10-02-traits.html)
+- [@4:27] - The book is for intermediate programming
+- [@5:32] - Most beneficial part about using Rust over other languages
+- [@7:42] - [Unreal Engine](https://www.unrealengine.com/)
+- [@11:13] - Unreal, [Unity](https://unity.com/) & [Godot](https://godotengine.org/)
+- [@13:44] - [Bevy](https://bevyengine.org/) Engine & [Amethyst](https://amethyst.rs/) Engine
+- [@18:31] - [Zig](https://ziglang.org/)
+- [@20:38] - Herbert created [Bracket-Lib](https://bracketproductions.com/) engine
+- [@24:18] - Steps to know when creating a game engine
+- [@31:20] - The book helps you with the concepts rather than the specifics
```

Comment:
This doesn't really seem like a section, just a statement. Can we rephrase this to capture what the next ~3 minutes were about?